doc/examples/reconst_gqi.py

```diff
@@ -127,7 +127,7 @@
 the same data and mask are used.
 """
 
-np.sum(gqpeaks.odf != ODF) == 0
+print(np.sum(gqpeaks.odf != ODF) == 0)
```

Comment:
I prefer that you remove the comment below and remove the `print`. Not sure this information is valuable.

Reply:
I think it is valuable as it proofs what is said in the immediately previous comment:
```
This ODF will be of course identical to the ODF calculated above as long as
the same data and mask are used.
```

So I'd keep it.

Reply:
ok, that's fine.